Lesson4/src/Homework4/Model/Person.java

```diff
@@ -47,7 +54,12 @@ public int setAge(int age) {
     }
 
     public String getName() {
-        return name;
+        name.trim();
+        String newName = name.substring(0,1).toUpperCase() + name.substring(1).toLowerCase();
```

Comment:
i think this should be done and verified in constructor, not in the getter, since we don't want to be able to create Person with wrong name